kamailio/default.cfg

```diff
@@ -150,7 +150,7 @@ loadmodule "xlog.so"
 
 ####### FIFO support for Management Interface  ########
 loadmodule "mi_fifo.so"
-modparam("mi_fifo", "fifo_name", "/tmp/kamailio_fifo")
+modparam("mi_fifo", "fifo_name", "/var/run/kamailio/kamailio_fifo")
```

Comment:
@swysor not sure on this, but if you remove the mod_param it should default to /var/run/kamailio/kamailio_fifo